src/Impostor.Api/Net/Inner/Objects/GameManager/IInnerHideAndSeekManager.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Impostor.Api.Net.Inner.Objects.GameManager.Logic.HideAndSeek;
+
+namespace Impostor.Api.Net.Inner.Objects.GameManager;
+
+public interface IInnerHideAndSeekManager
```

Comment:
does this interface need to inherit from IGameManager? idem IInnerNormalGameManager

Reply:
Solved.